server/sonar-server/src/test/java/org/sonar/server/rule/ws/AppActionTest.java

```diff
@@ -209,6 +209,51 @@ public void canWrite_is_false_if_user_is_not_profile_administrator_of_default_or
     assertJson(json).isSimilarTo("{ \"canWrite\": false }");
   }
 
+  @Test
+  public void canCustomizeRule_is_true_if_user_is_profile_administrator_of_default_organization_and_no_organization_is_specified() {
+    userSession.addPermission(OrganizationPermission.ADMINISTER_QUALITY_PROFILES, db.getDefaultOrganization());
+
```

Comment:
_minor code style_ 🐜 
These blank lines should probably separate test setup, test execution and assertions. If each of those parts is as short as in this case, the blank lines could be avoided in my opinion.